job-engine/app/web/src/main/java/org/eclipse/kapua/job/engine/app/web/jaxb/JaxbContextResolver.java

```diff
@@ -93,102 +34,19 @@
-                    JobEngineExceptionInfo.class,
-                    JobScopedEngineExceptionInfo.class,
-                    JobInvalidTargetExceptionInfo.class,
-                    JobMissingStepExceptionInfo.class,
-                    JobMissingTargetExceptionInfo.class,
-                    JobNotRunningExceptionInfo.class,
-                    JobResumingExceptionInfo.class,
-                    JobRunningExceptionInfo.class,
-                    JobStartingExceptionInfo.class,
-                    JobStoppingExceptionInfo.class,
-
-                    // Authentication
-                    AuthenticationXmlRegistry.class,
-                    AccessToken.class,
-
-                    // Device Management Keystore
-                    DeviceKeystores.class,
-                    DeviceKeystore.class,
-                    DeviceKeystoreCertificate.class,
-                    DeviceKeystoreItems.class,
-                    DeviceKeystoreItem.class,
-                    DeviceKeystoreItemQuery.class,
-                    DeviceKeystoreKeypair.class,
-                    DeviceKeystoreCSRInfo.class,
-                    DeviceKeystoreCSR.class,
-                    DeviceKeystoreXmlRegistry.class,
-
-                    // Jobs
-                    Job.class,
-                    JobQuery.class,
-                    JobXmlRegistry.class,
-
-                    JobStep.class,
-                    JobStepListResult.class,
-                    JobStepQuery.class,
-                    JobStepXmlRegistry.class,
-                    JobStepProperty.class,
-
-                    JobExecution.class,
-                    JobExecutionListResult.class,
-                    JobExecutionQuery.class,
-                    JobExecutionXmlRegistry.class,
-
-                    JobTarget.class,
-                    JobTargetListResult.class,
-                    JobTargetQuery.class,
-                    JobExecutionXmlRegistry.class,
-
-                    JobTargetSublist.class,
-
-                    DeviceCommandInput.class,
-                    DevicePackageDownloadRequest.class,
-                    DevicePackageDownloadOptions.class,
-                    DevicePackageInstallRequest.class,
-                    DevicePackageInstallOptions.class,
-                    DevicePackageUninstallRequest.class,
-                    DevicePackageUninstallOptions.class,
-                    DeviceAssets.class,
-                    DeviceConfiguration.class,
-
-                    Trigger.class,
-                    TriggerListResult.class,
-                    TriggerQuery.class,
-                    TriggerXmlRegistry.class,
-
-                    KuraDeviceConfiguration.class
-            }, properties);
-        } catch (Exception e) {
+            return KapuaLocator
+                    .getInstance()
+                    .getFactory(JAXBContextProvider.class)
+                    .getJAXBContext();
+        } catch (KapuaException e) {
```

Comment:
Instead of resolving the jaxb context in jetty and injecting it back in the KapuaLocator, we are fetching the one creating at guice's startup